irods-vfs-impl/src/main/java/org/irods/nfsrods/vfs/IRODSVirtualFileSystem.java

```diff
@@ -1645,15 +1663,101 @@ private static Inode toFh(long _inodeNumber)
         return Inode.forFile(Longs.toByteArray(_inodeNumber));
     }
 
+    private String tryAndFindPathFromGenQuery(long _inodeNumber, String option) {
+        // if getPath() returns null, we'll use GenQuery to find the path
+        log_.debug("""
+                vfs::tryAndFindPathFromGenQuery
+                tryAndFindPathFromGenQuery - _inodeNumber = {}
+                tryAndFindPathFromGenQuery - option       = {}""", 
+                _inodeNumber, option);
+
+        String strPath = null;
+        IRODSGenQueryBuilder builder;
+        IRODSGenQueryExecutor gqe;
+        IRODSGenQueryFromBuilder query;
+        IRODSQueryResultSet resultSet;
```

Comment:
Please initialize.

Reply:
Changed them all to be `null` at the start:

```java
IRODSGenQueryBuilder builder = null;
IRODSGenQueryExecutor gqe = null;
IRODSGenQueryFromBuilder query = null;
IRODSQueryResultSet resultSet = null;
```